app/script/util/DebugUtil.js

```diff
@@ -58,6 +58,39 @@ z.util.DebugUtil = class DebugUtil {
       .then(() => this.logger.log(`Corrupted Session ID '${sessionId}'`));
   }
 
+  haveISentThisMessageToMyOtherClients(messageId) {
+    const userId = wire.app.repository.user.self().id;
+    const conversation = wire.app.repository.conversation.active_conversation();
+    const conversationId = conversation.id;
+    const clientId = wire.app.repository.client.currentClient().id;
+    const message = conversation.get_message_by_id(messageId);
```

Comment:
Let's use the Promise of `get_message_in_conversation_by_id` in ConversationRepository. Then the message does not even have to be visible. All we need is conversation Id and message Id.